app/views/candidate_interface/sign_in/confirm_authentication.html.erb

```diff
@@ -1,16 +1,36 @@
-<% content_for :title, t('authentication.confirm_authentication.heading') %>
+<% if @authentication_token.user.last_signed_in_at.nil? %>
```

Comment:
I'd move this to a method (maybe application helper or a helper method in the controller) since is being called thrice. `if user_never_sign_in?` maybe?

Reply:
Good idea. I've added a `user.never_signed_in?` method.